internal/iam/command_rbac_role_binding_create.go

```diff
@@ -24,46 +24,46 @@ func (c *roleBindingCommand) newCreateCommand() *cobra.Command {
 	if c.cfg.IsCloudLogin() {
 		exs = append(exs,
 			examples.Example{
-				Text: `Grant the role "CloudClusterAdmin" to the principal "User:u-123456" in the environment "env-123456" for the cloud cluster "lkc-123456":`,
+				Text: "Grant the role `CloudClusterAdmin` to the principal `User:u-123456` in the environment `env-123456` for the cloud cluster `lkc-123456`:",
```

Comment:
We use quotes for resources and backticks for code snippets.
```suggestion
				Text: `Grant the role "CloudClusterAdmin" to the principal "User:u-123456" in the environment "env-123456" for the cloud cluster "lkc-123456":`,
```